streams/test-utils/src/test/java/org/apache/kafka/streams/test/MockProcessorContextStateStoreTest.java

```diff
@@ -155,7 +155,7 @@ public static Stream<Arguments> parameters() {
         return values.stream();
     }
 
-    @ParameterizedTest(name = "builder = {0}, timestamped = {1}, caching = {2}, logging = {3}")
+    @ParameterizedTest(name = "shouldEitherInitOrThrow(builder = {0}, timestamped = {1}, caching = {2}, logging = {3})")
```

Comment:
Could you please consider using `{displayName}` instead? This would help ensure naming consistency.

Additionally, we should address the following naming issues as well.

https://github.com/apache/kafka/blob/test-catalog/test-catalog/clients/tests.yaml#L817C3-L817C13
https://github.com/apache/kafka/blob/test-catalog/test-catalog/clients/tests.yaml#L3617
https://github.com/apache/kafka/blob/test-catalog/test-catalog/storage/tests.yaml#L30